components/status/StatusPreviewMastodon.vue

```diff
@@ -0,0 +1,137 @@
+<script setup lang="ts">
```

Comment:
running `pnpm run lint --fix` should format the files. I think it should have done it automatically 🤔

Reply:
It's running that command every time I commit, but it didn't pick this one up. Weird. Anywho, I manually deleted the superfluous variable assignment.